pkg/config/config.go

```diff
@@ -305,6 +305,8 @@ func initConfig(config Config) {
 	config.BindEnvAndSetDefault("logs_config.dev_mode_use_proto", true)
 	config.BindEnvAndSetDefault("logs_config.dd_url_443", "agent-443-intake.logs.datadoghq.com")
 	config.BindEnvAndSetDefault("logs_config.stop_grace_period", 30)
+	config.BindEnvAndSetDefault("logs_config.pipeline.count", 4)
+	config.BindEnvAndSetDefault("logs_config.pipeline.buffer_size", 100)
```

Comment:
Nit: I feel like pipeline count + buffer size is not as clear as channel count + channel size but that's just me.

Reply:
I think the idea is that a pipeline can have multiple channels and processors ?